content/news/rdkit-support-is-coming.md

```diff
@@ -64,11 +64,11 @@ This release will include a new backwards-incompatible [draft SMIRNOFF 1.0 speci
 * Eliminated "Requirements" section, since it specified requirements for the software, rather than described an aspect of the SMIRNOFF specification
 * Fractional bond orders are described in `<Bonds>`, since they currently only apply to this term.
 
-The current draft SMIRNOFF 1.0 specification can be found here: [Draft SMIRNOFF 1.0 specification](https://open-forcefield-toolkit.readthedocs.io/en/topology/smirnoff.html)
+The current draft SMIRNOFF 1.0 specification can be found here: [Draft SMIRNOFF 1.0 specification](https://github.com/openforcefield/openforcefield/blob/e3606e3aa2aefa14123e6179b6ece1cd35cf8f50/The-SMIRNOFF-force-field-format.md)
 
 ### New openforcefield `Molecule` and `Topology` objects
 
-Specifying the system to parameterize with [SMIRNOFF](https://open-forcefield-toolkit.readthedocs.io/en/topology/smirnoff.html) previously required passing along a set of OpenEye Toolkit [`OEMol`](https://docs.eyesopen.com/toolkits/python/oechemtk/OEChemClasses/OEMol.html) objects.
+Specifying the system to parameterize with [SMIRNOFF](https://github.com/openforcefield/openforcefield/blob/e3606e3aa2aefa14123e6179b6ece1cd35cf8f50/The-SMIRNOFF-force-field-format.md) previously required passing along a set of OpenEye Toolkit [`OEMol`](https://docs.eyesopen.com/toolkits/python/oechemtk/OEChemClasses/OEMol.html) objects.
```

Comment:
We keep legacy version of the docs around. How about https://open-forcefield-toolkit.readthedocs.io/en/0.2.0/smirnoff.html instead?

Reply:
Good idea!